lexicons/app/bsky/unspecced/applyLabels.json

```diff
@@ -0,0 +1,34 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.applyLabels",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Apply labels to a subject directly.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["subject"],
+          "properties": {
+            "subject": {
+              "type": "union",
+              "refs": [
+                "com.atproto.admin.defs#repoRef",
+                "com.atproto.repo.strongRef"
+              ]
+            },
+            "createLabelVals": {
```

Comment:
i kinda want this to just take already formatted labels 🤔 

so all the pds has to do is dumb shuffling of them off to the database, no formatting or anything & nowhere that the labels could get out of sync

Reply:
Yeah, I like that more—on it 👍